### ISSUE TITLE
Sampler for this object is not defined

### ISSUE THREAD

**Issue by AizazSharif** [ancorso/MAST#5]
Hi,

I am trying to run the current repo for example in pedestrians crosswalk. After installing all the packages, I am getting this error. 

![error](https://user-images.githubusercontent.com/9911205/94665022-f1221b80-030b-11eb-9af1-b899a7735a40.png)

The issue has been raised before but is closed now. Any help would be appreciated
Thanks.

**Comment by mossr** [comment]
This seems to be an issue with `Scene` not being sampleable, where `initialstate(mdp)` returns a `Scene` and the history recorder will try to sample an initial state.

A quick fix would be to add the following to the top of the "ped_crosswalk_with_blindspot.jl" file:
```julia
using Random
Base.rand(rng::AbstractRNG, s::Scene) = rand(rng, s.entities)
```
or
```julia
using Random
Base.rand(rng::AbstractRNG, s::Scene) = s
```
if it shouldn't sample an entity, but deterministically return the `Scene` (semi-hack at the moment).

But @ancorso should comment on whether this is a fix that should live in AdversarialDriving.jl or AutomotiveSimulator.jl.

**Comment by AizazSharif** [comment]
```julia
using Random
Base.rand(rng::AbstractRNG, s::Scene) = s
```
This was only working as one of the fixes you have mentioned above. The output is now:

![output](https://user-images.githubusercontent.com/9911205/94667807-8ffc4700-030f-11eb-9d9d-6f76a89c5a2d.png)

I wanted to ask if there is any other script to run for visualization of the crosswalk scenario.

**Comment by mossr** [comment]
I think running that file creates a "blindspot.gif" file, no?

**Comment by AizazSharif** [comment]
Yes, it does (my bad).

Thanks a lot for the help. Looking forward to contributing to this great work :)

**Comment by mossr** [comment]
No worries! Happy to help :)

**Comment by mossr** [comment]
I'll leave it open so we can resolve it for everyone else

**Comment by ancorso** [comment]
Thanks for tracking the issue down @mossr. 

It looks like this new requirement stems from a POMDPs.jl v0.9, where `initialstate(mdp)` should return a distribution. I'll make the needed change.

**Comment by ancorso** [comment]
Fixed by #6
